libsolidity/formal/CHC.cpp

```diff
@@ -692,6 +710,63 @@ void CHC::visitAddMulMod(FunctionCall const& _funCall)
 	SMTEncoder::visitAddMulMod(_funCall);
 }
 
+void CHC::visitDeployment(FunctionCall const& _funCall)
+{
+	if (!m_settings.externalCalls.isTrusted())
+	{
+		SMTEncoder::endVisit(_funCall);
+		return;
+	}
+
+	auto funType = dynamic_cast<FunctionType const*>(_funCall.expression().annotation().type);
```

Comment:
```suggestion
	auto const* funType = dynamic_cast<FunctionType const*>(_funCall.expression().annotation().type);
```

Reply:
I'd prefer to leave it out here since it's already described explicitly in the cast